person.rb

```diff
@@ -1,28 +1,23 @@
 require_relative 'nameable'
-require_relative 'rental'
 
 class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id, :rentals
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
-    @id = Random.rand(1..1000)
+    @id = Random.rand(1..50)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @rentals = []
   end
 
-  private
-
-  def of_age?
-    true if @age >= 18
-    false
```

Comment:
- [ ] After creating a person, checking the list of people does not show said person, this is a bug that has been introduced with the new commits, kindly fix this.

Before commits
![image](https://github.com/Ademola-coding/School-Library/assets/81932478/b446f023-6ba2-4f9d-b2b2-6cfc96f640d6)

After
![image](https://github.com/Ademola-coding/School-Library/assets/81932478/f6cad4f4-442b-4830-9a8a-d0cbf100f16f)

Reply:
Bugs identified and are all fixed

Reply:
**ADDING PERSON**

![Screenshot (96)](https://github.com/Ademola-coding/School-Library/assets/96092850/259be645-07e0-4705-9c5b-e37ef2d1b444)

![Screenshot (97)](https://github.com/Ademola-coding/School-Library/assets/96092850/f3f27062-cdca-49e8-b889-1d85a91e889d)

**PERSON ADDED To JSON FILE**
![Screenshot (98)](https://github.com/Ademola-coding/School-Library/assets/96092850/7babbdbc-ec44-4b33-a056-717bd32d973e)